tests/gh-actions/install_KinD_create_KinD_cluster_install_kustomize.sh

```diff
@@ -54,8 +58,11 @@ nodes:
   image: kindest/node:v1.32.2@sha256:f226345927d7e348497136874b6d207e0b32cc52154ad8323129352923a3142f
 - role: worker
   image: kindest/node:v1.32.2@sha256:f226345927d7e348497136874b6d207e0b32cc52154ad8323129352923a3142f
-" | kind create cluster --config -
+" | kind create cluster --name=${KIND_CLUSTER_NAME} --config - --wait 120s
```

Comment:
```suggestion
" | kind create cluster --config - --wait 120s
```

why do we need the name?